lib/ui/blessed/element.js

```diff
@@ -94,7 +94,7 @@ blessed.Element.prototype.setLabel = function (options) {
       self._label.rtop = self.childBase || 0;
     }
     // PATCH BEGIN
-    if (!self._label.detached) self.screen.render();
+    // if (!self._label.detached) self.screen.render();
```

Comment:
IIRC, it has been added for a good reason :) I need to dig into the history but I remember label was not repositioned when scrolling.

Reply:
I thought so too, but it seems to work fine without it? 

Conceptually, I'm not sure I see a reason to trigger another screen `render` when there is a static `label` on a `scrollable` box (i.e it doesn't move, so why `render` the whole screen again).

However, if you do notice a difference I'll dig into it.

Reply:
For example, if you select an event, scroll to the bottom, then select another event, the label is not positioned correctly.

Reply:
Nice catch! I've added a commit that should fix this! Let me know if it's OK on your side